src/App.tsx

```diff
@@ -14,210 +30,109 @@ export const App: React.FC = () => {
     }
   }, []);
 
+  useEffect(() => {
+    async function todosFromServer(userId: number | undefined) {
+      try {
+        const visibleTodos = getTodos(userId);
+
+        setTodos(await visibleTodos);
+      } catch (error) {
+        setErrorMessage(`${error}`);
+      }
+    }
+
+    if (!user) {
+      return;
+    }
+
+    todosFromServer(user.id);
+  }, []);
+
+  const filteredTodos = useMemo(() => {
+    return todos.filter(todo => {
+      switch (filterBy) {
+        case FilterBy.All:
+          return todo;
+
+        case FilterBy.Active:
+          return !todo.completed;
+
+        case FilterBy.Completed:
+          return todo.completed;
+
+        default:
+          return null;
+      }
+    });
+  }, [todos, filterBy]);
+
+  const handleTodos = async (event: React.FormEvent) => {
+    event.preventDefault();
+
+    if (!title) {
+      setErrorMessage('Title can`t be empty');
+
+      return;
```

Comment:
*Optional 
its possible to write as `return setErrorMessage('Title can`t be empty');`